_docker/drupal/drupal-filesystem/web/config/sync/core.entity_view_display.node.article.rss.yml

```diff
@@ -5,17 +5,33 @@ dependencies:
   config:
     - core.entity_view_mode.node.rss
     - field.field.node.article.body
+    - field.field.node.article.field_article_highlights
```

Comment:
@jordanpagewhite These extra fields here expected? Looks like changes to a content type.

cheers,

Rob

Reply:
Yes, these are all expected. I am not sure why they were not already in config, but there shouldn't be an issue with adding them now. They should be here since they're all hidden fields on this view mode.